Assets/PatchKit/UnityEditorExtension/Editor/Core/Config.cs

```diff
@@ -253,7 +253,7 @@ public static void UnlinkApp(AppPlatform platform)
         }
 
         EditorUtility.SetDirty(instance);
-        AssetDatabase.SaveAssets();
+        //AssetDatabase.SaveAssets(); //this create error 
```

Comment:
What error? Need a better explanation than this.

Reply:
Also, if something is causing problems, it should be removed instead of commented out (the only exception is disabling something temporary until the further fix, but in such case, there should be always `TODO:` or `FIX:` comment to mark this place in the code).

Reply:
Error : 

> GUI Window tries to begin rendering while something else has not finished rendering! Either you have a recursive OnGUI rendering, or previous OnGUI did not clean up properly.

> device.IsInsideFrame()

I think we don't need this because file Config remove secret and save file without this line

Reply:
That's not always the case, we need to call this line in order to make sure that the config is saved.

Can you paste the stack trace of this error?
Most probably, `UnlinkApp` is called somewhere without `Dispatch` that makes sure, that function is called outside of OnGUI function and does not interrupt its process.

Reply:
Ok I used Dispatch and this works, but a bug has arisen, Unity is minimized when I press Save All

Reply:
Probably an error Unity Editor